inst/extdata/installPackages.R

```diff
@@ -0,0 +1,62 @@
+# Install repgen, production instance, prerequisite packages. Should be safe to
+# run repeatedly without additional intervention.
+
+pkgs <- c(
+  "dplyr",
+  "evaluate",
+  "htmlTable",
+  "jsonlite",
+  "lazyeval",
+  "knitr",
+  "lubridate",
+  "rmarkdown",
+  "yaml"
+)
+
+# convenience wrapper function around install.packages()
+installPackages <- function(pkgs, lib, repos = getOption("repos")) {
+  for (p in pkgs) {
+    tryCatch({
+      packageDescription(p) # check to see if package p is already installed
+    },
+    warning = function(w) {
+      # if package p is not installed...
+      if (any(grepl("(DESCRIPTION file of package .+ is missing or broken|no package .+ was found)", w))) {
+        # ...install it
+        install.packages(p, lib, repos = repos)
+      } else {
+        print(w)
+      }
+    },
+    error = function(e) {
+      print(e)
+    })
+  }
+}
+
+lib <- Sys.getenv("R_LIBS")
+if (nchar(lib) == 0) {
+  stop("Could not get a value for R_LIBS environment variable")
+}
+
+# all packages except devtools and its prerequisites are held back to older
+# versions
+installPackages(pkgs, lib, "http://mran.microsoft.com/snapshot/2016-03-31")
```

Comment:
No reason not to use HTTPS. I just copy-and-pasted the URL from the older installer code.